conans/model/ref.py

```diff
@@ -89,23 +89,20 @@ def __new__(cls, name, version, user, channel, revision=None, validate=True):
         @param channel:     string containing the user channel
         @param revision:    string containing the revision (optional)
         """
-        version = Version(version)
+        version = Version(version) if version is not None else None
         obj = super(cls, ConanFileReference).__new__(cls, name, version, user, channel, revision)
         if validate:
-            obj.validate()
+            obj._validate()
         return obj
 
-    def validate(self):
+    def _validate(self):
         ConanName.validate_name(self.name, reference_token="package name")
         ConanName.validate_name(self.version, True, reference_token="package version")
         ConanName.validate_name(self.user, reference_token="user name")
         ConanName.validate_name(self.channel, reference_token="channel")
         if self.revision:
             ConanName.validate_revision(self.revision)
 
-    def __hash__(self):
```

Comment:
I think I made this, can't recall why but for a reason. Have you removed trying? or you know certainly that it is no longer needed?

Reply:
Maybe force a big build.

Reply:
Fired new build with svn & slown and revisions=1. but can't fire the big, big one.

https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-4166/4/pipeline/